### PR TITLE
Тягунов -- Лабораторная работа #1

### PR DESCRIPTION
**Pull request by vtyagunov**

### CODE REVIEW

**Comment by vtyagunov** [comment]
@AndrewSmrnv @KalyanovDO review, please

File: code/polygon-area/src/test/java/ru/unn/agile/polygon/model/LineTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        Line line = new Line(p1, p2);
+
+        assertEquals(p2.getX() - p1.getX(), line.getX(), delta);
+        assertEquals(p2.getY() - p1.getY(), line.getY(), delta);
+
+        assertEquals(p1.getX(), line.getXPoint1(), delta);
+        assertEquals(p1.getY(), line.getYPoint1(), delta);
+
+        assertEquals(p2.getX(), line.getXPoint2(), delta);
+        assertEquals(p2.getY(), line.getYPoint2(), delta);
```
Comment (KalyanovDO) [inline code/polygon-area/src/test/java/ru/unn/agile/polygon/model/LineTest.java]: Could you please use one assert per one test in all your tests?

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java [hunk 2]
```diff
@@ -1,6 +1,8 @@
-package ru.unn.agile.ComplexNumber.model;
+package ru.unn.agile.ComplexNumber.Model;
```
Comment (KalyanovDO) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: You can't change any files in this repo except your own, please pay attention.

File: code/polygon-area/src/test/java/ru/unn/agile/polygon/model/LineTest.java (hunk 1 above)
Comment (vtyagunov) [reply code/polygon-area/src/test/java/ru/unn/agile/polygon/model/LineTest.java]: done.

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java (hunk 2 above)
Comment (vtyagunov) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: Gradle can't be built without that change

**Comment by AndrewSmrnv** [comment]
👍

**Comment by KalyanovDO** [comment]
:+1:

**Pull request merged**